src/Razor/src/Microsoft.AspNetCore.Razor.Language/RazorDiagnosticSeverity.cs

```diff
@@ -6,6 +6,7 @@ namespace Microsoft.AspNetCore.Razor.Language
     public enum RazorDiagnosticSeverity
     {
         // Purposely using the same value as Roslyn here.
+        Warning = 2,
```

Comment:
Curious: If you build in VS do you get a warning in the window?

Reply:
Nope. Not even an error. Build just succeeds.